test/integration/testpaths/testpath_migration_matrix.py

```diff
@@ -17,7 +17,8 @@
 """ Test cases for Delta Snapshots Test Path
 """
 import time
-from marvin.lib.base import ServiceOffering, Configurations, VirtualMachine, Account, Volume, DiskOffering, StoragePool
+from marvin.lib.base import ServiceOffering, Configurations, VirtualMachine, Account, Volume, DiskOffering, StoragePool, \
+    Role
```

Comment:
I think there is a more need line for a multiline import in python (/me googling ...)
```suggestion
from marvin.lib.base import (ServiceOffering,
                             Configurations,
                             VirtualMachine,
                             Account,
                             Volume,
                             DiskOffering,
                             StoragePool,
                             Role)
```